v3/benchmarking/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/kwasnick/circuit/v3/benchmarking
+
+go 1.16
+
+require (
+	github.com/afex/hystrix-go v0.0.0-20180502004556-fa1af6a1f4f5
+	github.com/cep21/circuit/v3 v3.2.0
```

Comment:
One problem here, the benchmarking won't change if we update the library code without updating the tag here :(

Reply:
You're right, that is unfortunate. I'm not 100% sure on how this works, but I believe the `replace` directive with a relative path would work to fix this

```
replace github.com/cep21/circuit/v3 => ../
```

Reply:
It looks like it works, based on what gets put in go.mod and go.sum with the replace directive https://github.com/cep21/circuit/pull/109/commits/37cce18431d7dcc3c95b70faafc270d07df271b1